string md5sum
string definition
string datatype
int32 uncompressed_size
byte[] data
